apps/docs/guide/getting-started.md

```diff
@@ -29,131 +30,107 @@ $ pnpm install @rtco/client
 ```sh [bun]
 $ bun add @rtco/client
 ```
+
 :::
 
+## Setup
 
-## Call Example
+### Basic Setup
 
-The following example shows how to connect two peers and share audio/video/data between them:
+```js
+import { Artico } from "@rtco/client";
+const rtco = new Artico();
+```
 
-#### Peer 1
+### Setup with Custom ID
 
-```ts
-import { Artico } from "@rtco/client";
+::: warning
+Creating custom ID with the Out-of-the-box Signaling can cause ID conflicts. This will return a bad request error (400) when it happens
+:::
 
-const rtco = new Artico();
+```js
+import { Artico } from "@rtco/client";
+import { v4 } from "uuid";
 
-rtco.on("open", (id: string) => {
-  // We're now connected to the signaling server. `id` refers to the unique ID that
-  // is currently assigned to this peer, so remote peers can connect to us.
-  console.log("Connected to signaling server with peer ID:", id);
+const rtco = new Artico({
+  id: v4.slice(0, 5), // creates a 5 characted ID
 });
+```
```

Comment:
I'm not sure if this is a good way to express the idea of how to use this `id`.
In fact, I'd like the docs to say something like:
> You can request a specific peer ID. If you do, Artico will atempt to register such ID with the signaling server. If not available, Artico will emit an `Error` with the message `"invalid-id"`.
> **Warning:** It is recommended that you let Artico assign you a random ID, since IDs should be universally unique within Artico's signaling network.

I haven't fully tested this, but taking a quick look at the code, you should be able to capture when an ID is in use by doing something like:

```ts
const rtco = new Artico({id: 'duplicate'});

rtco.on("error", (err: Error) => {
  if (err.message === 'id-taken') {
    console.log("ID is taken!");
  }
})
```

Reply:
I've tried it now and it works, but it still throws a bad request error. I have even wrapped it in a try catch and still throws..

Reply:
Oh, the constructor throws? I see... I  can fix that in the next release.

Reply:
@AnupamKris I've just release v0.2.5 with the fixes. It shouldn't throw anymore and you should be just able to cleanly detect the `id-taken` error.